### ISSUE TITLE
Custom ¨Thank you¨ page redirect?

### ISSUE THREAD

**Issue by lobau** [pentacent/keila#322]
Our landing page has a custom Thank you page https://volu.dev/thankyou (which we currently use for SendGrid)

I would love to move to Keila, but right now the Thank you page I see is not great, and not customizable.
Here is the code I am using

```html
<form target="_blank" class='newsletter-form' action="https://app.keila.io/forms/nfrm_xxxxxxxx" method="post">
    <input id="contact_email" name="contact[email]" placeholder="Your email address" type="email" required/>
    <button type="submit">Subscribe</button>
</form>
```
Is there a way to customize where the user lands after submitting?

**Comment by wmnnd** [comment]
Thank you for this suggestion! I think this should be a relatively easy and useful add. I’ll see if I can get to it at some point this week :blush:

**Comment by lobau** [comment]
Awesome, I love to hear this!

A quick prior art search:

This is formspree.io setting page, per form:
![image](https://github.com/user-attachments/assets/ca0538f5-b601-403b-95fd-a176de81fe51)

Mailchimp settings
![image](https://github.com/user-attachments/assets/fb731091-60b0-4204-949f-332709625d94)
https://mailchimp.com/help/design-and-host-your-own-thank-you-pages/

Hope it helps 😊

**Comment by wmnnd** [comment]
Implemented in fbc750e. 
![image](https://github.com/user-attachments/assets/61f7858a-a41b-4cfb-a8d1-223e7f9c3ce8)
